test/json/json.test.js

```diff
@@ -13,6 +15,22 @@ describe('json flag', () => {
         expect(parseJson).not.toThrow();
     });
 
+    it('should store json to a file', (done) => {
+        const { stdout } = run(__dirname, ['--json', 'stats.json']);
+
+        expect(stdout).toContain('stats are succesfylly storerd as json to stats.json');
+        stat(resolve(__dirname, './stats.json'), (err, stats) => {
+            expect(err).toBe(null);
+            expect(stats.isFile()).toBe(true);
+            done();
+        });
+        readFile(resolve(__dirname, 'stats.json'), 'utf-8', (err, data) => {
+            expect(err).toBe(null);
+            expect(() => JSON.parse(data)).not.toThrow();
```

Comment:
Also add tests for few keys in parsed JSON.

Reply:
We should check these in object returned by `JSON.parse(data)`

Reply:
done